test/common/router/config_impl_test.cc

```diff
@@ -2031,6 +2034,221 @@ TEST(RoutePropertyTest, excludeVHRateLimits) {
   EXPECT_TRUE(config_ptr->route(headers, 0)->routeEntry()->includeVirtualHostRateLimits());
 }
 
+TEST(CustomRequestHeadersTest, AddNewHeader) {
+  const std::string json = R"EOF(
+  {
+    "virtual_hosts": [
+      {
+        "name": "www2",
+        "domains": [
+          "lyft.com",
+          "www.lyft.com",
+          "w.lyft.com",
+          "ww.lyft.com",
+          "wwww.lyft.com"
+        ],
+        "request_headers_to_add": [
+          {
+            "key": "x-client-ip",
+            "value": "%CLIENT_IP%"
+          }
+        ],
+        "routes": [
```

Comment:
i do not think that all those routes are important for this test purpose. Can you make that shorter?